conf/app.src

```diff
@@ -0,0 +1,6 @@
+SOURCE_URL=https://github.com/duniter/cesium/releases/download/v1.3.11/cesium-v1.3.11-web.zip
```

Comment:
Do you plan to maintain this link each time a new version is released?
It would fix #10, but it would require more maintenance, than current behaviour.

Reply:
That is the idea, yes. Maybe not "Pre-release" but all "Latest release" from https://github.com/duniter/cesium/releases
The idea of Yunohost is to be easy for server owner. So far, Cesium was not updated by them as new versions were not appearing in the admin panel.
The difficulty will come more from 
 SOURCE_SUM=fcac385f959178fc4ef8ac323b482c7d618bc48f67d6cc189229f91629b144be
as it means downloading the zip and get the SUM...
I guess it can be done. How often @blavenie release a new version?
Anyway, yunohost owner won't update cesium everyday. Maybe I should ask @blavenie if there are a sort of major release and smaller ones.

Reply:
You can add an helper (a script) in the repository to ease the upgrade and the calculation of the checksum. That script could download last zip using the line removed in this PR, calculate the checksum and change the `SOURCE_SUM` variable with `sed`.

> How often @blavenie release a new version?

You can check the repository.

---

duniter_ynh is a little bit a pain in the ass to maintain, since GitLab migrationthere is no longer a unique url to retrieve last release from an URL. So, if you plan to maintain it, I won’t block you.

Reply:
I'll try my best. I'll make an other PR for that helper.

Reply:
I'll try my best.
I've made an other PR for that helper. I wrote a comment to get help on making it because json is new and sed is new for me...

https://github.com/YunoHost-Apps/cesium_ynh/pull/15